tool/template-404.html

```diff
@@ -0,0 +1,23 @@
+{% extends "template-base.html" %}
+
+{% block left_sidebar %}
+{% set use_page = pages|selectattr("html", 'defined_and_equalto', "docs.html")|first %}
+{% include 'template-sidebar_nav.html' %}
```

Comment:
This awkwardly doesn't actually show the headings for "Concept" / "Tutorial" / "Reference" above their component categories, but it does provide a reasonably useful & thorough list of pages. Actually fixing the thing to show those would be a fair bit harder (probably involving dozens of new lines in the sidebar_nav template) so I decided to go with something faster for the first iteration of this.

Reply:
That seems fine to me for the first go. We can always circle back later to polish it. (Famous last words, but I'm sure we'll encounter more broken links in the future.)